### PR TITLE
Fc 2127 doc examples of tailwind

### PR DESCRIPTION
**Pull request by mGGk-fr**
# What

This introduce a guide to setup tailwind in a front-commerce project

# Preview
https://deploy-preview-894--heuristic-almeida-1a1f35.netlify.app/docs/3.x/guides/extend-vite-configuration/tailwind

### CODE REVIEW

File: docs/02-guides/extend-vite-configuration/_category_.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+label: Extend the Vite Configuration
+collapsible: true
+link:
+  type: generated-index
+  description:
+    This section contains goal-oriented documentation that provides step-by-step
+    instructions to extend the Vite configuration with popular extensions.
```
Comment (paul-vd) [inline docs/02-guides/extend-vite-configuration/_category_.yml:7]: Although this was in the ticket, i'm not sure what this has to do with the vite configuration.

File: docs/02-guides/extend-vite-configuration/tailwind.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+description:
+  This guide explains how to setup Tailwind in your Front-Commerce application
+  thanks to Vite.
+---
+
+<p>{frontMatter.description}</p>
+
+:::tip
+
+You can check the `example-extension/tailwind` folder from the skeleton to check
```
Comment (paul-vd) [inline docs/02-guides/extend-vite-configuration/tailwind.mdx:12]: Add link

Comment (paul-vd) [inline docs/02-guides/extend-vite-configuration/tailwind.mdx:12]:
```suggestion
You can check the `example-extension/tailwind` example to see how tailwind can be implemented.
```

Possibly link to the readme?

File: docs/02-guides/extend-vite-configuration/tailwind.mdx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
++ content: ["./app/**/*.{ts,tsx,js,jsx}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+} satisfies Config
+```
+
+If you have components in some of your extensions, you need to add the related
+extensions' paths to the `content` directive.
```
Comment (paul-vd) [inline docs/02-guides/extend-vite-configuration/tailwind.mdx:47]:
```suggestion
If you have components in some of your extensions, you need to add the related
extension paths to the `content` directive.
```

File: docs/02-guides/extend-vite-configuration/tailwind.mdx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+## Setup PostCSS
+
+The `tailwindcss init` command should have generated a `postcss.config.js` file,
+you can remove this file and add the `tailwind` configuration to the
+`postcss.config.mjs` file instead:
```
Comment (paul-vd) [inline docs/02-guides/extend-vite-configuration/tailwind.mdx:53]: Demonstrate the tailwindcss init command.

Remember, this docs goes off of the bases that they have not seen the README.md in the example extension, so I think it should duplicate a lot of those docs.

File: docs/02-guides/extend-vite-configuration/tailwind.mdx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+```
+
+Now you can head to
+[`http://localhost:4000/tailwind`](http://localhost:4000/tailwind) and check
+that Tailwind is working.
+
+## More infos
+
+If you want more infos about Tailwind setup with Remix, please check
+[this guide](https://tailwindcss.com/docs/guides/remix).
```
Comment (paul-vd) [inline docs/02-guides/extend-vite-configuration/tailwind.mdx:124]: I don't think this deserves a title, rather at the start of this docs add a references section with link to tailwind and remix

```suggestion
## References

- https://tailwindui.com/documentation
- https://tailwindcss.com/docs/guides/remix
```

**Review by paul-vd — CHANGES_REQUESTED** [review]

File: docs/02-guides/extend-vite-configuration/_category_.yml (hunk 1 above)
Comment (mGGk-fr) [reply docs/02-guides/extend-vite-configuration/_category_.yml:7]: The implementation of tailwind is made thanks to vite, i think it's more something like "this things that you can do with front-commerce can be done thanks to vite"

Comment (real34) [reply docs/02-guides/extend-vite-configuration/_category_.yml:7]: Exactly. The goal is to show common use cases for Vite specific configuration.

**Review by paul-vd — APPROVED** [review]

**Comment by real34** [comment]
@mGGk-fr this should have targeted `main`. Can you port it to `main` please? (I don't think we have 3.5 specific doc on `next` already, so it could be only merging `next` into `main`.

**Pull request merged**